tools/bazel

```diff
@@ -19,35 +19,39 @@
 # supported version, and then calling it. This way, we can make sure
 # that running bazel will always get meaningful results, at least
 # until Bazel 1.0 is released.
+# NOTE: This script relies on bazel's feature where //tools/bazel
+# script can be used to hijack "bazel" invocations in given workspace.
 
 set -e
 
 VERSION=0.24.1
 
-CWD=`pwd`
+echo "INFO: Running bazel wrapper (see //tools/bazel for details), bazel version $VERSION will be used instead of system-wide bazel installation."
+
+CWD=$(pwd)
```

Comment:
Nit: This doesn't look ideal. Could we use a subshell with a single `cd` at the beginning? Or `pushd` and `popd`?